### PR TITLE
Add myfritz.net

### PR DESCRIPTION
**Pull request by juliushacker**

### CODE REVIEW

**Comment by hkamran80** [comment]
Is this different from the site that already exists? The other MyFRITZ! entry is in the `remote` category.

**Review by hkamran80 — CHANGES_REQUESTED** [review]
This site doesn't appear to be different from the site that already exists in the `remote` category.

**Comment by juliushacker** [comment]
You're right, it's the same site. I didn't see the original entry under sso.myfritz.net.

As sso.myfritz.net isn't the primary domain of the service, I suggest labeling the entry under the domain myfritz.net with additional domain sso.myfritz.net

Additionally I fixed the existing entry as the service doesn't provide U2F.

File: img/m/myfritz.net.svg [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+	<path class="st49" d="M21.1,8.5V7c0.6,0,2.3,0,4.7,0.1l-2.5,4.6c0.5,0,1.3,0,2.3,0l0,1.6c0,0-4.7,0-4.8,0l-0.1,0l2.5-4.8
+		C22.6,8.5,22,8.5,21.1,8.5z"/>
+	<polygon class="st50" points="7.5,8 7.5,8.5 5.6,8.5 5.6,9.9 7.3,9.9 7.3,11.4 5.7,11.4 5.7,13.8 3.8,13.8 3.8,7.1 7.5,7 	"/>
+	<polygon class="st49" points="26.5,11.1 27.7,11.1 28.1,8.1 26.7,7.9 	"/>
+	<g>
+		<polygon class="st50" points="26.2,12.1 27.1,11.3 27.8,12.2 26.9,13 26.2,12.1 		"/>
+		<path class="st50" d="M27,12.1"/>
+	</g>
+</g>
+</svg>
```
Comment (hkamran80) [inline img/m/myfritz.net.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 60 60"><path fill="#fee702" d="m25.575 6.855-9.76 8.654-6.443 5.913-2.884 2.549-.817.576.24 10.097 10.192 9.615 7.212 6.443 2.356 2.067 1.058-.049 10.722-9.23 7.355-6.923 1.73-1.539.529-3.847 1.203-6.345-6.54-5.144L28.46 7.96l-1.394-1.01-.771-.307z"/><path fill="#231f20" d="M26.182 6.437c-.376.03-.673.232-.834.343-.386.265-2.042 1.594-4.293 3.604a259.207 259.207 0 0 0-6.383 5.933 513.175 513.175 0 0 1-4.935 4.627l.01.012h-.24l.004-1.245-.118-.308s-4.634-.015-9.043.175l-.35.015v16.814h1.71v.836h4.383l.001-2.162c.552.531 3.372 3.247 4.89 4.666.47.44 4.644 4.345 6.517 6.026.477.428 1.06.963 1.684 1.533 1.398 1.28 2.983 2.73 3.991 3.562l.455.375c1.16.957 1.64 1.353 1.913 1.525.25.158.502.238.757.238.104 0 .21-.017.315-.043.124.082.484.318.654.423.206.128.501.355 1.182-.05.679-.403 1.97-1.35 4.147-3.27a394.884 394.884 0 0 0 6.571-5.98c1.439-1.34 4.55-4.285 5.92-5.615a283.63 283.63 0 0 0 2.616-2.578h6.273v-3.43l1.717 2.12.813-.658.61.753 2.224-1.8-2.174-2.686 1.7.064L60 22.922l-.99-.078.146-.946-4.295-.34-.5 8.268 1.414.052-2.348 1.901h-.424v-1.221l-.465.026c-1.31.086-2.418.113-3.362.116l5.159-9.6c-.526-.03-1.032-.056-1.531-.079l.78-1.45-.644-.045c-5.454-.309-9.473-.25-11.063-.25-.04-.037-4.555-4.249-6.824-6.274-2.28-2.033-4.314-3.833-5.366-4.63-1.053-.797-1.288-.992-1.554-1.063a3.053 3.053 0 0 0-.332-.064c-.536-.403-.845-.6-.868-.614a1.232 1.232 0 0 0-.751-.194zm.1.41a.84.84 0 0 1 .432.138c.008.004.834.529 2.153 1.649.352.297.709.596 1.069.895 1.011.844 2.057 1.716 2.99 2.612.687.66 1.806 1.66 2.888 2.63.929.83 1.805 1.616 2.376 2.155a317.682 317.682 0 0 0 3.277 3.017v2.96H29.303V21.59l-.97.007v-1.586l-3.817.003.559 2.267-1.757.438.022 12.463h1.717l.001.828h4.245v-8.619c.232-.006.538-.013.87-.023v.774h2.387V38.72h1.716v.83h4.327V28.142h4.104V24.62c1.295-.058 2.292-.089 3.105-.105l-5.466 10.548h2.384l-.43.83h2.917c-.564.558-2.674 2.637-5.558 5.344-3.292 3.089-12.15 10.835-12.756 11.201-.535.322-.9.175-1.158.012-.252-.16-.76-.579-1.874-1.499l-.454-.375c-1.001-.824-2.582-2.272-3.977-3.548-.624-.57-1.21-1.107-1.687-1.536-1.518-1.364-4.807-4.425-6.511-6.02-.208-.194-.5-.473-.756-.715h1.619v.844h4.2v-5.997l3.397 5.64.318-.264c.126-.103.248-.208.372-.31l.643 1.066c1.21-.999 3.444-3.1 3.444-3.1a193.72 193.72 0 0 0-1.227-1.386l-2.424-2.844c1.837-1.52 2.303-2.676 2.202-4.02-.088-1.162-.578-1.946-1.267-2.482-.838-2.107-3.293-2.545-5.011-2.75-1.63-.185-4.228-.072-6.007.003l-.268.01.008-2.215h-.145c.508-.472 3.185-2.955 4.62-4.335.8-.769 3.622-3.468 6.372-5.923 2.237-1.997 3.873-3.31 4.251-3.57.247-.17.468-.28.698-.276zM6.043 24.578l4.36.004V38.68a875.432 875.432 0 0 1-4.31-4.121l.003-3.162h4.022v-3.47l-.98.005.002-1.645-3.1.014zm10.26 2.9h.653c.087.18.144.387.144.626 0 .634-.33 1.227-.797 1.764z"/><path fill="#ed1c24" fill-rule="evenodd" d="M43.33 19.645c-.676.001-1.202.005-1.561.004v3.735c2.137-.097 3.73-.087 5.1-.098l-5.91 11.404.263.001h11.416v-3.735c-2.5.164-4.345.115-5.632.085l5.988-11.144c-4.255-.24-7.634-.257-9.663-.252zm-34.186.123-8.774.175V36.04h4.382l.004-5.846h4.021l.002-3.47-4.083.02.007-3.368 4.445-.007-.004-1.342zm18.818.62-2.98.006.55 2.16-1.835.458.02 11.796h4.245zm27.251 1.571-.453 7.507 2.84.105 1.13-7.333zm-26.382 1.317v3.757l4.104-.093v11.406h4.327V26.94l4.104.093v-3.757zm-14.244.134c-1.304-.005-2.722.057-3.824.102V38.4h4.2V26.275h.904c.421 0 1.592.587 1.592 1.828 0 1.312-1.177 2.402-2.143 3.164l4.48 7.438c1.211-.999 2.36-1.984 3.364-3.018l-3.76-4.41c1.912-1.558 2.392-2.727 2.29-4.092-.218-2.885-2.912-3.432-4.976-3.677a20.374 20.374 0 0 0-2.127-.098zm41.578 6.632-2.224 1.8 1.802 2.223 2.223-1.8z"/></svg>
```

Here's a smaller SVG

File: entries/m/myfritz.net.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "MyFRITZ!Net": {
+    "domain": "myfritz.net",
+    "url": "https://www.myfritz.net",
```
Comment (hkamran80) [inline entries/m/myfritz.net.json]:
```suggestion
```

The `url` key is only needed if going to the domain (e.g. `https://myfritz.net`) fails

File: entries/m/myfritz.net.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "additional-domains": [
+      "sso.myfritz.net"
+    ],
```
Comment (hkamran80) [inline entries/m/myfritz.net.json]:
```suggestion
```

You don't need to add subdomains if the base domain is there

**Comment by phallobst** [comment]
The use of U2F is actually supported, as I was just able to confirm with my YubiKey. The login screen allows for "Confirmation with One-time Password" (= TOTP) or "Confirmation with Proof of Personal Identity" (= U2F) if these methods are set up.

File: entries/m/myfritz.net.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    "tfa": [
+      "totp",
```
Comment (hkamran80) [inline entries/m/myfritz.net.json:5]:
```suggestion
      "totp",
      "u2f",
```

Their documentation says they support U2F security keys

File: entries/m/myfritz.net.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+      "custom-hardware"
+    ],
+    "custom-hardware": [
+      "Smartphone biometrics (fingerprint, face recognition)"
+    ],
```
Comment (hkamran80) [inline entries/m/myfritz.net.json:11]: How are the biometrics read? Is it through an app that they have on the Play Store and/or App Store?

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/m/myfritz.net.json (hunk 5 above)
Comment (juliushacker) [reply entries/m/myfritz.net.json:11]: I just rechecked that - it seems they only use Webauthn to support hardware tokens as well as smartphone biometrics.

**Comment by juliushacker** [comment]
@phallobst Thanks for clearing the point that they mean U2F with their second option. I misunderstood the configuration option on the site, as the description there just is about biometrics.

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**